specification/_types/common.ts

```diff
@@ -253,6 +254,48 @@ export enum NodeStatsLevel {
   shards
 }
 
+export enum Metrics {
+  adaptive_selection,
+  allocations,
+  breaker,
+  discovery,
+  fs,
+  http,
+  indexing_pressure,
+  indices,
+  ingest,
+  jvm,
+  os,
+  process,
+  repositories,
+  thread_pool,
+  transport
+}
```

Comment:
This enum is not used. But I already defined it as NodeStatsMetric in specification/nodes/stats/NodesStatsRequest.ts anyway, with a few values that were missing in the 8.x docs so far.